client/utils.lua

```diff
@@ -64,10 +66,85 @@ if GetConvarInt('ox_target:drawSprite', 1) == 1 then
             end
         end
 
+
+
         return n > 0 and inRange, newZone
     end
 
+
+    function utils.getNearbyTarget(coords)
+        coords = coords or GetEntityCoords(PlayerPedId())
+        inRange = {}
+
+        local n = 0
+        local targetData  = getTargetData();
+        local activated   = getActivated();
+        local isActive    = state.isActive();
+
+        local getDistance = function(obj) 
+            return #(GetEntityCoords(PlayerPedId())-GetEntityCoords(obj))
```

Comment:
And now we call PlayerPedId and GetEntityCoords for every single entity known to the client, on every single frame.